spec/index.bs

```diff
@@ -393,6 +393,13 @@ dictionary GPUExtensions {
 <script type=idl>
 dictionary GPULimits {
     unsigned long maxBindGroups = 4;
+    unsigned long maxDynamicUniformBuffersPerPipelineLayout = 8;
+    unsigned long maxDynamicStorageBuffersPerPipelineLayout = 4;
+    unsigned long maxSampledTexturesPerShaderStage = 16;
+    unsigned long maxSamplersPerShaderStage = 16;
```

Comment:
is that really addressing the confusion about bind groups? We should still say "PerBindGroup", I think, for clarity.

Reply:
Actually, these limits apply to the entire pipeline layout as well. So by definition a single bind group must not exceed these limits, but when we combine bind group layouts into pipeline layouts, their combined totals will have to be validated against these same limits.

Saying "per bind group" is also true, but then wouldn't make as much sense as a limit for `GPUPipelineLayout` validation.

Reply:
Some of these apply to descriptors per stage, some apply to all descriptors in the entire layout. The one you quoted is for general Uniform buffers, but the one that applies to pipeline layout is DYNAMIC uniform buffers. When validating the descriptor for a dynamic uniform buffer, my understanding is that both of these limits have to be checked separately. Did I understand that correctly?

Reply:
Ok, now as I re-read it, it makes more sense. So we have shader stages and bind groups as two orthogonal things. The are two kinds of limits here: one kind that is a total across all shader stages and bind groups (XxxPerPipelineLayout), and one that is a total across all bind groups of *each* shader stage (XxxPerShaderStage).